### DISCUSSION TITLE
Audio Mix Tool for volume control on all open streams

### DISCUSSION THREAD

**Discussion by rigbymatt** [f1multiviewer/issue-tracker#528] [Feature requests] [answered]
Would be great to have a basic window with vertical volume sliders similar to a mixing board or DAWs like Ableton, Pro Tools or Mixing Station on iPad, with one volume control or "fader" per stream, all controlling the same parameter as the volume sliders on each streams individual window but in one centralized location, along with audio output selection, and maybe L/R panning.

Some examples of mix windows:

https://mixingstation.app/
https://www.mixinglessons.com/wp-content/uploads/2018/03/Customise-mix-window-in-pro-tools-first-minimal.png
https://ableton-production.imgix.net/live-manual/11/SessionViewMixer.png?auto=compress%2Cformat&w=762
https://static.packt-cdn.com/products/9781849693004/graphics/3004OT_7_3.jpg

**Comment by ExtremTechniker** [comment]
Nice Idea

**Comment by f1multiviewer** [comment, marked as answer]
Since MultiViewer's API allows you to change the volume programatically, I think this might be a perfect thing for a plugin to fix. This isn't something I imagine a lot of people would use, so is outside of our scope for the app.
